sentry/sentry.conf.example.py

```diff
@@ -185,32 +185,32 @@ def get_internal_network():
 SENTRY_WEB_HOST = "0.0.0.0"
 SENTRY_WEB_PORT = 9000
 SENTRY_WEB_OPTIONS = {
-    "http-keepalive": True,
+    "http": "%s:%s" % (SENTRY_WEB_HOST, SENTRY_WEB_PORT),
+    "protocol": "uwsgi",
+    # This is need to prevent https://git.io/fj7Lw
+    "uwsgi-socket": None,
     "so-keepalive": True,
-    "http-auto-chunked": True,
+    "http-keepalive": 5,
```

Comment:
Relay keeps connections for 15 (min) to 75 (max) seconds. Might want to align this?

Reply:
Done. I read Apache 2.2+ reduced this to 5, hence the 5 second limit here but I'll align it with Relay.